src/producer.h

```diff
@@ -82,6 +82,11 @@ class Producer : public Connection {
 
   void ConfigureCallback(const std::string &string_key, const v8::Local<v8::Function> &cb, bool add) override;
 
+  Baton InitTransactions(int32_t timeout_ms);
+  Baton BeginTransaction();
+  Baton CommitTransaction(int32_t timeout_ms);
+  Baton AbortTransaction(int32_t timeout_ms);
```

Comment:
SendOffsetsToTransaction() is also required

Reply:
I am not familiar with committing offsets, it is something I need to investigate. Do you think it is possible to merge this PR and leave that for future work?